.travis.yml

```diff
@@ -0,0 +1,9 @@
+group: travis_latest
+
+git:
+  depth: 5
+
+language: clojure
+lein: 2.8.1
```

Comment:
This requires a decision on the lein version to support. I would suggest the latest, since the more you support the more it becomes complicated. It is also possible to do a matrix build with various lein versions if needed.